### DISCUSSION TITLE
Unable to test Notifications

### DISCUSSION THREAD

**Discussion by titantwentyone** [filamentphp/filament#3757] [Help]
Was looking to test that a notification fired from an action and saw that `assertNotified` is available.

When I ran this, I got:

```
Error : Call to undefined function Filament\Notifications\Testing\expect()
```
This is an old project and I do plan on migrating everything over to Pest at some stage but it's not a priority at the moment. This assertion assumes Pest is being used and the reason for the test falling over. The docs on testing also state that:

> All examples in this guide will be written using [Pest](https://pestphp.com/). However, you can easily adapt this to a PHPUnit.

v3 should probably use Pest exclusively but perhaps v2 should still support PHPUnit?

**Comment by danharrin** [comment]
This is a bug, we shouldn't be relying on Pest internally. Sorry about that. Will PR a fix.

**Comment by danharrin** [comment]
Fixed by #3811.

**Reply by titantwentyone** [reply]
Thank you!
